diff --git a/src/dpp/queues.cpp b/src/dpp/queues.cpp
index 9d9d0355..37e29893 100644
--- a/src/dpp/queues.cpp
+++ b/src/dpp/queues.cpp
@@ -146,7 +146,7 @@ http_request_completion_t http_request::run(cluster* owner) {
 		};
 	} else {
 		/* Always attach token and correct user agent when sending REST to Discord */
-		headers.emplace("Authorization", "Bot " + owner->token);
+		headers.emplace("Authorization", owner->token);
 		headers.emplace("User-Agent", http_version);
 		if (!reason.empty()) {
 			headers.emplace("X-Audit-Log-Reason", reason);
